<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="main_widget">
   <property name="styleSheet">
    <string notr="true">

QPushButton {
	border-radius: 8px;
	margin-left: 10px;
	margin-right: 10px;
	padding-top: 5px;
	padding-bottom: 5px;
	padding-left: 15px;
	padding-right: 15px;
	border: 1px solid #ffffff;
	background-color: rgba(255, 255, 255, 120); 
	color: black;  
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 150); /* немного менее прозрачный при наведении */
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 192); /* чуть плотнее при нажатии */
}

QWidget, QLabel, QPushButton {
    color: #111;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="list_widget" native="true">
      <property name="styleSheet">
       <string notr="true">#list_widget	{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(255, 0, 57, 255), stop:1 rgba(3, 0, 255, 255));
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTabWidget" name="tab_widget">
         <property name="styleSheet">
          <string notr="true">QTabWidget {
	border: 0px solid #ffffff;
	/* Это устанавливает полупрозрачный фон для области содержимого QTabWidget */
	background-color: rgba(255, 255, 255, 120); 
}

QTabWidget::pane {
	border: 0px solid #ffffff;
    background: transparent;
    
}

QTabBar::tab {
	/* Устанавливаем полупрозрачный фон для вкладок в нормальном состоянии */
	background: rgba(255, 255, 255, 150); 
	border: 1px solid #ffffff; /* Можно добавить тонкую белую границу для видимости */
	margin-right: 2px;
	padding: 6px 12px;
	border-top-left-radius: 6px;
	border-top-right-radius: 6px;
}

QTabBar::tab:selected {
	/* Устанавливаем полупрозрачный или непрозрачный фон для выбранной вкладки */
	background: rgba(0, 0, 255, 200); /* Полупрозрачный синий */
	/* Или полностью непрозрачный: background: #0000ff; */
	color: white;
}

QTabBar::tab:hover {
	/* Устанавливаем полупрозрачный фон при наведении */
	background: rgba(51, 0, 255, 180); 
}
</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_lists">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="accessibleName">
           <string>dawd</string>
          </property>
          <attribute name="icon">
           <iconset>
            <normaloff>../icons/to-do-list.png</normaloff>../icons/to-do-list.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Списки</string>
          </attribute>
          <layout class="QVBoxLayout" name="layout_lists">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="page_hide_lists">
          <attribute name="icon">
           <iconset>
            <normaloff>../icons/to-do-list_2.png</normaloff>../icons/to-do-list_2.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Скрытые списки</string>
          </attribute>
          <layout class="QVBoxLayout" name="layout_hide_lists">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="config_widget" native="true">
      <property name="styleSheet">
       <string notr="true">#config_widget	{
	border: 1px solid #ffffff;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(3, 0, 255, 255), stop:1 rgba(9, 0, 57, 255));
}</string>
      </property>
      <layout class="QVBoxLayout" name="layout_buttons" stretch="0,0,0,0,0">
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="but_create_list">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Создать список</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/edit_2.png</normaloff>../icons/edit_2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="but_learned_words">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Изученные слова</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/files.png</normaloff>../icons/files.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="but_setting">
         <property name="text">
          <string>Настройки</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/setting.png</normaloff>../icons/setting.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="profile_widget" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget {

	border-radius: 10px;
	margin: 5px;
	border: 1px solid #ffffff;
	background-color: rgba(246, 245, 244, 100);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_user_name">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	background-color: rgba(0, 0, 0, 0);
	border: none;
}</string>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="midLineWidth">
             <number>2</number>
            </property>
            <property name="text">
             <string>User Name</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>1</number>
            </property>
            <property name="indent">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_desc">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;</string>
            </property>
            <property name="text">
             <string>Изученных слов:</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="but_quantity_learned_words">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton	{
	border: none;
	text-align: right;
	padding-right: 5px;
	margin-right: 10px;
}</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icons/refresh_2.png</normaloff>../icons/refresh_2.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="but_logout">
            <property name="styleSheet">
             <string notr="true">#but_logout {
	margin-top: 10px;
	margin-bottom: 10px;
	margin-right: 20px;
	margin-left: 20px;
}</string>
            </property>
            <property name="text">
             <string>Выйти</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icons/exit.png</normaloff>../icons/exit.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
